app/code/Magento/Quote/Model/CustomerManagement.php

```diff
@@ -3,14 +3,18 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
 
 namespace Magento\Quote\Model;
 
-use Magento\Customer\Api\CustomerRepositoryInterface as CustomerRepository;
 use Magento\Customer\Api\AccountManagementInterface as AccountManagement;
 use Magento\Customer\Api\AddressRepositoryInterface as CustomerAddressRepository;
-use Magento\Quote\Model\Quote as QuoteEntity;
+use Magento\Customer\Api\CustomerRepositoryInterface as CustomerRepository;
+use Magento\Customer\Model\AddressFactory;
 use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Validator\Exception;
```

Comment:
Could you import it as ValidatorException? It will not be clear that `throw new Exception` is not the same as `throw new \Exception`